source/includes/elastic-agents/_031-job-completion.md

```diff
@@ -0,0 +1,70 @@
+## Job Completion
```

Comment:
Add link to https://github.com/gocd/plugin-api.go.cd/blob/master/source/includes/elastic-agents/_015-plugin-request-messages.md

Reply:
None of the other ones have it too. We can add a reference to all the other requests too

Reply:
All of them are linked in that page

Reply:
Well, I check all the new extension docs has it.

Reply:
Please, can you add the request name and link in the above-mentioned file?